### ISSUE TITLE
Allow `time` column to be configurable

### ISSUE THREAD

**Issue by yatharthranjan** [RADAR-base/RADAR-JDBC-Connector#11]
The time column for create table in TimescaleDB dialect needs to be configurable. https://github.com/RADAR-base/RADAR-JDBC-Connector/blob/4534bad49cf322c5e1c9b17baf1b96ef2becffd6/kafka-connect-jdbc/src/main/java/io/confluent/connect/jdbc/dialect/TimescaleDBDatabaseDialect.java#L90

While most of the raw data in RADAR does have the `time` column, the data coming out of KSQL always has uppercase columns, so `time` is converted to `TIME` (unless you quote it every time it is used and in every query which is less than ideal).

Relevant logs-
```
[2021-05-18 11:50:36,103] INFO Creating table with sql: [CREATE SCHEMA IF NOT EXISTS public, CREATE TABLE "public"."QUESTIONNAIRE_CAT_SCORE_STREAM" (
"timestamp" BIGINT NOT NULL,
"PROJECTID" TEXT NOT NULL,
"USERID" TEXT NOT NULL,
"SOURCEID" TEXT NULL,
"TIME" TIMESTAMPTZ NOT NULL,
"TIMECOMPLETED" DOUBLE PRECISION NULL,
"TIMENOTIFICATION" DOUBLE PRECISION NULL,
"NAME" TEXT NULL,
"VERSION" TEXT NULL,
"CAT_SCORE" INT NULL,
PRIMARY KEY("PROJECTID","USERID","TIME")), SELECT create_hypertable('"public"."QUESTIONNAIRE_CAT_SCORE_STREAM"', 'time', migrate_data => TRUE, chunk_time_interval => INTERVAL '1 day');] (io.confluent.connect.jdbc.sink.DbStructure)
[2021-05-18 11:50:36,129] WARN Create failed, will attempt amend if table already exists (io.confluent.connect.jdbc.sink.DbStructure)
org.postgresql.util.PSQLException: ERROR: column "time" does not exist
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2510)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2245)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:311)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:309)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:295)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:272)
	at org.postgresql.jdbc.PgStatement.executeUpdate(PgStatement.java:246)
	at io.confluent.connect.jdbc.dialect.GenericDatabaseDialect.applyDdlStatements(GenericDatabaseDialect.java:1181)
	at io.confluent.connect.jdbc.dialect.TimescaleDBDatabaseDialect.applyDdlStatements(TimescaleDBDatabaseDialect.java:105)
	at io.confluent.connect.jdbc.sink.DbStructure.create(DbStructure.java:115)
	at io.confluent.connect.jdbc.sink.DbStructure.createOrAmendIfNecessary(DbStructure.java:61)
	at io.confluent.connect.jdbc.sink.BufferedRecords.add(BufferedRecords.java:123)
	at io.confluent.connect.jdbc.sink.JdbcDbWriter.write(JdbcDbWriter.java:77)
	at io.confluent.connect.jdbc.sink.JdbcSinkTask.put(JdbcSinkTask.java:74)
	at org.apache.kafka.connect.runtime.WorkerSinkTask.deliverMessages(WorkerSinkTask.java:545)
	at org.apache.kafka.connect.runtime.WorkerSinkTask.poll(WorkerSinkTask.java:325)
	at org.apache.kafka.connect.runtime.WorkerSinkTask.iteration(WorkerSinkTask.java:228)
	at org.apache.kafka.connect.runtime.WorkerSinkTask.execute(WorkerSinkTask.java:200)
	at org.apache.kafka.connect.runtime.WorkerTask.doRun(WorkerTask.java:184)
	at org.apache.kafka.connect.runtime.WorkerTask.run(WorkerTask.java:234)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

The solution would be to make the time key configurable in the properties and can be changed according to requirements.

**Comment by blootsvoets** [comment]
As a first heuristic, the statement matching the `time` column  could be made case insensitive? In any case, that's pretty weird behaviour from KSQL..

**Comment by mpgxvii** [comment]
Yes we could either:
1. Make `time` column configurable in properties file
2. Make `time` case insensitive
3. Make `time` column configurable for each topic in properties file

**Comment by blootsvoets** [comment]
Option 1 will not work for all topics. Alternative: have multiple JDBC connectors running at the same time, one for a given `time` column value. For example:

connector 1: acceleration, magnetic_field: time column.
connector 2: ksql_model: TIME column.

The option 3 is the most general, but will need some weird properties-file parsing. I'd prefer option 2 or 4.

**Comment by yatharthranjan** [comment]
Option 2 sounds good then. Option 1 is also feasible for us. Yes, we were planning the same to have one connector for aggregate data and the other for raw data.

**Comment by mpgxvii** [comment]
Solved in #12
